### ISSUE TITLE
Crash when attempting placing underwater torch on the bottom of a block

### ISSUE THREAD

**Issue by sunshinekitsune** [SirSquidly/Oceanic-Expanse#107]
If you try to place an underwater torch on the bottom of a block, instead of doing nothing the game crashes.
```
java.util.concurrent.ExecutionException: java.lang.IllegalArgumentException: Cannot set property PropertyDirection{name=facing, clazz=class net.minecraft.util.EnumFacing, values=[up, north, south, west, east]} to down on block oe:underwater_torch, it is not an allowed value
    at java.util.concurrent.FutureTask.report(FutureTask.java:122)
    at java.util.concurrent.FutureTask.get(FutureTask.java:191)
    at net.minecraft.util.Util.runTask(SourceFile:531)
    at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:723)
    at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:668)
    at net.minecraft.server.integrated.IntegratedServer.tick(IntegratedServer.java:185)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:1817)
    at java.lang.Thread.run(Thread.java:1570)
Caused by: java.lang.IllegalArgumentException: Cannot set property PropertyDirection{name=facing, clazz=class net.minecraft.util.EnumFacing, values=[up, north, south, west, east]} to down on block oe:underwater_torch, it is not an allowed value
    at net.minecraft.block.state.BlockStateContainer$StateImplementation.withProperty(BlockStateContainer.java:227)
    at com.sirsquidly.oe.blocks.BlockUnderwaterTorch.getStateForPlacement(BlockUnderwaterTorch.java:85)
    at net.minecraft.block.Block.getStateForPlacement(Block.java:2273)
    at net.minecraft.item.ItemBlock.onItemUse(ItemBlock.java:52)
    at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:889)
    at net.minecraft.item.ItemStack.onItemUse(ItemStack.java:190)
    at net.minecraft.server.management.PlayerInteractionManager.processRightClickBlock(PlayerInteractionManager.java:481)
    at net.minecraft.network.NetHandlerPlayServer.processTryUseItemOnBlock(NetHandlerPlayServer.java:741)
    at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.processPacket(SourceFile:55)
    at net.minecraft.network.play.client.CPacketPlayerTryUseItemOnBlock.processPacket(SourceFile:11)
    at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
    at java.util.concurrent.FutureTask.run(FutureTask.java:317)
    at net.minecraft.util.Util.runTask(SourceFile:529)
```

**Comment by SirSquidly** [comment]
Checking this, it is caused by how poorly done my code for Underwater Torch placement is currently, they were one of the first blocks added to the mod. I have a rewrite of their placement already done, which will also solve this issue.

**Comment by SirSquidly** [comment]
Fixed in 1.1.5
